developer-docs-site/docs/move/book/vector.md

```diff
@@ -157,22 +157,11 @@ fun destroy_droppable_vector<T: drop>(vec: vector<T>) {
 }
 ```
 
-Similarly, vectors cannot be copied unless the element type has `copy`. In other words, a
-`vector<T>` has `copy` if and only if `T` has `copy`. However, even copyable vectors are never
-implicitly copied:
-
-```move
-let x = vector::singleton<u64>(10);
-let y = copy x; // compiler error without the copy!
-```
-
-Copies of large vectors can be expensive, so the compiler requires explicit `copy`'s to make it
-easier to see where they are happening.
+Similarly, vectors cannot be copied unless the element type has `copy`. 
```

Comment:
```suggestion
Similarly, vectors cannot be copied unless the element type has `copy`. In other words, a
`vector<T>` has `copy` if and only if `T` has `copy`.
```

I think if we do change this, we should at least leave this part in, it's a little clearer!

Reply:
Get it back. 
By the way, are you the Matt I talked with in Discord 😀?